MonstercatNet/Models/Playlist/Playlist.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SoftThorn.MonstercatNet.Models.Playlist
```

Comment:
I'd like to cut down the namespace to the root namespace, so that users of this lib have to only import a single namespace to get all the available features.

Reply:
@Insire Which root namespace? I noticed that all the other models are under directories inside the Models directory.
E.g. Release, Self, Track.

Where would you like me to move the `Playlist` class?

Reply:
no no, dont move the class. its fine where it is, just shorten the namespace from ``namespace SoftThorn.MonstercatNet.Models.Playlist`` to ``namespace SoftThorn.MonstercatNet``